### DISCUSSION TITLE
Default Values when Adding Products

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#451] [Ideas]
Hi Team,

When adding every product, we have to choose the value for these every time, it will be better if you give the option to set default somewhere and so people don't have to choose every time and they make mistakes.

1) Barcode Type
2) Searchable
3) Stock Management Enabled

![image](https://user-images.githubusercontent.com/16856248/147846636-d53a8c26-6310-4960-a0e6-de474781e792.png)

Also, not everyone has a wholesale price. So you can give an option to use sale price in wholesale price by default and then , allow them to override if required.

Like the option, used in checkout, 

Shipping Address same as Billing Address, if they uncheck then they can override it. Like wise.

**Comment by Blair2004** [comment]
We rather think about cloning products so those value are defined on clone.
